lib/src/main/java/io/hcxprotocol/createresource/HCXInsurancePlan.java

```diff
@@ -0,0 +1,489 @@
+package io.hcxprotocol.createresource;
+
+import ca.uhn.fhir.model.api.annotation.*;
+import ca.uhn.fhir.model.api.annotation.Extension;
+import ca.uhn.fhir.util.ElementUtil;
+import org.hl7.fhir.r4.model.*;
+
+@ResourceDef(name="InsurancePlan", profile="https://ig.hcxprotocol.io/v0.7/StructureDefinition-HCXInsurancePlan.html")
+public class HCXInsurancePlan extends InsurancePlan {
+
+    @Block
+    public static class InsurancePlanPlanComponent extends InsurancePlan.InsurancePlanPlanComponent{
```

Comment:
@saurabh-singh-17 - do you think should we use HCX as prefix for all HCX custom classes?